demos/README.md

```diff
@@ -1,6 +1,11 @@
 # Open Model Zoo Demos
 
-The Open Model Zoo demo applications are console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use-cases
+The Open Model Zoo demo applications are console applications that demonstrate how you can use the Inference Engine in your applications to solve specific use-cases.
+
+For the Intel® Distribution of OpenVINO™ toolkit, the demos are available after installation in the folder: `<INSTALL_DIR>\deployment_tools\open_model_zoo\demos`.
+For the open source version of OpenVINO™ toolkit, the demos are available in the Open Model Zoo repository in the `demos` directory.
```

Comment:
I'm not sure this sentence makes sense. What do you mean by the open source version of the toolkit?

Reply:
I think it reflect the version that you may build ourselves from github dldt repo

Reply:
Yes, open sourced is the repository on GitHub, which provides the source code
https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos

Demos folder in the Intel distribution contains what you need to run the demo when getting started

Reply:
Okay, but this sentence still doesn't sit right with me. The demos are available in the OMZ repository regardless of which version of OpenVINO toolkit is used. Also, saying "the Open Model Zoo repository" doesn't help those who don't know where this repository is, while those who _do_ know don't need any help (I imagine the conclusion that demos are in the `demos` directory is pretty obvious).

Maybe something like "The demos can also be obtained from the [Open Model Zoo repository](https://github.com/openvinotoolkit/open_model_zoo/)." would work better.

Reply:
ok, changed